File: Toggly.FeatureManagement.NET/Toggly.FeatureManagement/Data/JsonWebKeySet.cs
using System.Collections.Generic;

namespace Toggly.FeatureManagement.Data
{
    /// <summary>
    /// JSON Web Key Set
    /// </summary>
    public class JsonWebKeySet
    {
        /// <summary>
        /// JWK array from a JWKS document; may be null when absent or not yet deserialized.
        /// </summary>
        public List<JsonWebKey>? Keys { get; set; }
    }
}

File: Toggly.FeatureManagement.PHP/src/Toggly/FeatureManagement/Models/JsonWebKeySet.php
<?php

namespace Toggly\FeatureManagement\Models;

/**
 * JSON Web Key Set
 */
class JsonWebKeySet
{
    /**
     * List of JSON Web Keys
     * @var JsonWebKey[]
     */
    public array $keys = [];

    public function __construct(array $data = [])
    {
        if (isset($data['keys'])) {
            $this->keys = array_map(function ($key) {
                return $key instanceof JsonWebKey ? $key : new JsonWebKey($key);
            }, $data['keys']);
        }
    }
}
